src/test/java/org/takes/facets/hamcrest/HmRsStatusTest.java

```diff
@@ -49,11 +49,11 @@
     public void responseCodeIsEqualToExpected() throws IOException {
```

Comment:
@erimerturk we should rename this method `testsStatusOK`. Also, the javadoc for the method should start with `HmRsStatus can...`, so let's do `HmRsStatus can test status HTTP_OK`